src/backend/distributed/deparser/deparse_view_stmts.c

```diff
@@ -157,8 +158,13 @@ AppendAlterViewCmd(StringInfo buf, AlterTableCmd *alterTableCmd)
 
 		case AT_ColumnDefault:
 		{
-			elog(ERROR, "Citus doesn't support setting or resetting default values for a "
-						"column of view");
+			if (EnablePropagationWarnings)
```

Comment:
why do we ever need to deparse a non-distributed view such that we have to change `AppendAlterViewCmd`?

Reply:
It is also left behind from start. Tests pass without it, so I removed the check.